docs/software/mqtt/index.mdx

```diff
@@ -62,6 +74,20 @@ An example of a received `NODEINFO_APP` message:
 }
 ```
 
+The meaning of these fields is as follows:
+
+- "`id`" is the unique ID for this message. 
+- "`channel`" is the channel index this message was received on.
+- "`from`" is the unique node number of the node on the mesh that sent this message. 
```

Comment:
Worth mentioning the existing bug where in the JSON this intended-to-be-unsigned number is interpreted as a signed number (https://github.com/meshtastic/firmware/issues/2565)?

As this seems to be the go-to stable identifier for a specific node, it seems likely to be used and relied on. Seeing a different value via protobuf and JSON feels likely to yield confusion.

Reply:
Added in https://github.com/meshtastic/meshtastic/pull/746/commits/740ac50764f94fcfb5a40ece8c279366da524a3f